cou/steps/plan.py

```diff
@@ -34,18 +36,42 @@ def generate_plan(args: Namespace) -> UpgradeStep:
     return plan
 
 
+def prompt(parameter: str) -> str:
+    """Generate eye-catching prompt."""
+
+    def bold(text: str) -> str:
+        return colored(text, "red", attrs=["bold"])
+
+    def normal(text: str) -> str:
+        return colored(text, "red", attrs=["blink"])
+
+    return (
+        normal(parameter + " (")
+        + bold("c")
+        + normal(")ontinue/(")
+        + bold("a")
+        + normal(")bort/(")
+        + bold("s")
+        + normal(")kip:")
+    )
+
+
 def apply_plan(upgrade_plan: Any) -> None:
     """Apply the plan for upgrade."""
-    result = input(upgrade_plan.description + "[Continue/abort/skip]")
-    if result.casefold() == "c".casefold():
-        upgrade_plan.run()
-    elif result.casefold() == "a".casefold():
-        sys.exit(1)
-    else:
-        logging.info("Skipped")
-
-    for sub_step in upgrade_plan.sub_steps:
-        apply_plan(sub_step)
+    result = "X"
+    while result not in "CcAsSs":
```

Comment:
I suggest to make this as constant + you made mistake it should be `"CcAaSs"`.
However we can do better with:
```python
while result := input(prompt(upgrade_plan.description)).lower() not in "cas":
    match result:
        case "c":
            upgrade_plan.run()
                for sub_step in upgrade_plan.sub_steps:
                    apply_plan(sub_step)
        case "a":
            logging.info("Aborning plan")
            sys.exit(1)
        case "s":
            logging.info("Skipped")
        case _:
            logging.info("No valid input provided!")
```

Reply:
If we enter c,a or s it never runs

Reply:
fixed `CcAaSs` issue

Reply:
done